jdk-bootstrap/src/main/java/com/github/rmee/jdk/bootstrap/JdkBootstrapExtension.java

```diff
@@ -2,6 +2,8 @@
 
 public class JdkBootstrapExtension {
 
+	private String vendor;
```

Comment:
One possible trap I can see with this is that adoptopenjdk is the default in the constructor - so if someone calls setUrlTemplate, they might forget to also set a vendor. Do we want to automatically set it to "unknown" or something if they call that?